components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -952,6 +1022,28 @@ def getAnnotation(self, ns=None):
             return AnnotationWrapper._wrap(self._conn, rv[0].child, link=rv[0])
         return None
 
+    def getAnnotationCount(self):
```

Comment:
This could be named `countAnnotations`. I would also expect `ns` as a param to match `listAnnotations`